python/lsst/analysis/tools/atools/calibQuantityProfile.py

```diff
@@ -0,0 +1,358 @@
+            3: "C03",
+            4: "C04",
+            5: "C05",
+            6: "C06",
+            7: "C07",
+            8: "C10",
+            9: "C11",
+            10: "C12",
+            11: "C13",
+            12: "C14",
+            13: "C15",
+            14: "C16",
+            15: "C17",
+        }
+
+        self.prep.panelKey = "amplifier"
+        self.prep.dataKey = "mjd"
+
+    def finalize(self):
+        super().finalize()
+
+        # Configure each panel
+        for key, value in self.produce.plot.valsGroupBy.items():
+            gridPanelConfig = GridPanelConfig(
+                plotElement=self.plotElement,
+                title={"label": str(value), "fontsize": "10"},
+                titleY=0.85,
+            )
+            self.produce.plot.panels[key] = gridPanelConfig
+            self.produce.plot.panels[key].plotElement.xKey = f"{value}_x"
+            self.produce.plot.panels[key].plotElement.valsKey = value
+
+
+class CalibDivisaderoScatterTool(CalibQuantityBaseTool):
+    def setDefaults(self):
+        super().setDefaults()
+        self.plotElement = ScatterElement()
+
+        # Repack the input data into a usable format
+        self.prep = SingleValueRepacker()
+
+        self.produce.plot = GridPlot()
+        self.produce.plot.panels = {}
+        self.produce.plot.numRows = 2
+        self.produce.plot.numCols = 1
+
+        # Values to group by to distinguish between data in differing panels
+        self.produce.plot.valsGroupBy = {0: "bank1", 1: "bank0"}
+
+        self.produce.plot.xDataKeys = {
+            0: "bank1_x",
+            1: "bank0_x",
+        }
+        self.prep.panelKey = "amplifier"
+        self.prep.dataKey = "mjd"
+
+    def finalize(self):
+        super().finalize()
+
+        # Configure each panel
+        for key, value in self.produce.plot.valsGroupBy.items():
+            gridPanelConfig = GridPanelConfig(
+                plotElement=self.plotElement,
+                title={"label": str(value), "fontsize": "10"},
+                titleY=0.85,
+            )
+            self.produce.plot.panels[key] = gridPanelConfig
+            self.produce.plot.panels[key].plotElement.xKey = f"{value}_x"
+            self.produce.plot.panels[key].plotElement.valsKey = value
+
+
+class CalibPtcCovarScatterTool(CalibQuantityBaseTool):
+    def setDefaults(self):
+        super().setDefaults()
+        self.plotElement = ScatterElement()
+
+        # Repack the input data into a usable format
+        self.prep = PrepRepacker()
+
+        self.produce.plot = GridPlot()
+        self.produce.plot.panels = {}
+        self.produce.plot.numRows = 3
+        self.produce.plot.numCols = 2
+
+        # Values to group by to distinguish between data in differing panels
+        self.produce.plot.valsGroupBy = {
+            0: "PTC_COV_10",
+            1: "PTC_COV_20",
+            2: "PTC_COV_01",
+            3: "PTC_COV_02",
+            4: "PTC_COV_11",
+            5: None,
```

Comment:
What does the 5: None do?

Reply:
Grid plot needs to be able to generate an N*M grid of panels, but I only need 5, and 5 is unfortunately prime.  By setting this to `None`, no columns will match, and the sixth panel will be empty.